src/controllers/groupsController.js

```diff
@@ -73,9 +74,9 @@ const getGroup = async (req, res) => {
     group.owner._id !== req.user.userID &&
     !group.members.some((member) => member._id === req.user.userID)
   ) {
-    console.log('ownerID:', group.owner._id);
+    console.log("ownerID:", group.owner._id);
     console.log(group.members.some((member) => member._id === req.user.userID));
-    throw new Error('You do not have permission to access this group.');
+    throw new Error("You do not have permission to access this group.");
```

Comment:
Thanks for adding this, @marice-romero! 🎉 

This works, but another option would be to filter the group selection, similarly to what you did with the update endpoint. That way we don't bother loading the data if we don't need to. But I get that might have been tricky to write since you want to any group where member id OR owner id equals the requesting user id. I think `Group.where` would work for this. (No need for adjustments...just FYI. 😉 )